doc/examples/denoise_localpca.py

```diff
@@ -0,0 +1,117 @@
+"""
+===============================
+Denoise images using Local PCA
+===============================
+
+Using the local PCA based denoising for diffusion
```

Comment:
Can you rewrite this paragraph? A bit confusing - perhaps it will be nice to give some more information of the PCA denoising techniques.